### PR TITLE
add unifuncnet

### PR DESCRIPTION
**Pull request by PedroMTQ**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by PedroMTQ** [comment]
https://github.com/orgs/conda-forge/teams/staged-recipes Can you review this PR please?

**Comment by PedroMTQ** [comment]
@conda-forge/staged-recipes Can you review this PR please?

File: recipes/unifuncnet/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{% set name = "unifuncnet" %}
+{% set version = "1.0.2" %}
+
```
Comment (goanpeca) [inline recipes/unifuncnet/meta.yaml:3]: Could you remove the extra spaces?

File: recipes/unifuncnet/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  host:
+    - setuptools >=42
+    - wheel
+    - pip
+    - python >=3.9
+  run:
+    - python >=3.9
+    - requests
+    - beautifulsoup4 
+    - lxml
```
Comment (goanpeca) [inline recipes/unifuncnet/meta.yaml:31]: Sorting the list alphabetically makes it easier to read :) 

For example:

```yaml
requirements:
  host:
    - pip
    - python >=3.9
    - setuptools >=42
    - wheel
  run:
    - beautifulsoup4 
    - lxml 
    - python >=3.9
    - requests
```

Comment (PedroMTQ) [reply recipes/unifuncnet/meta.yaml:27]: I added your proposed changes

**Review by goanpeca — APPROVED** [review]

**Comment by PedroMTQ** [comment]
Thank you for the approval.

When would this be merged and added to the conda-forge channel?

**Comment by PedroMTQ** [comment]
@goanpeca 
Hello,
Could you let me know how long it should take to create the feedstock? The previous tool I added was a bit quicker so I was wondering if there's something I would need to do?

Thanks,
Pedro

File: recipes/unifuncnet/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    - beautifulsoup4 
+    - lxml 
+    - python >=3.9
+    - requests
+    
+test:
+  imports:
+    - unifuncnet
+  commands:
+    - pip check
```
Comment (beenje) [inline recipes/unifuncnet/meta.yaml:35]: It would be nice to test the entry point:
```suggestion
    - pip check
    - unifuncnet --help
```

**Pull request merged**